BUILDING.rst

```diff
@@ -0,0 +1,19 @@
+Building the Documentation
+==========================
+
+In order the build the documentation, for example to convert the ``*.rst`` files into HTML files,
+you will need to run `make` at the command line, along with a suitable target. The build process
+also requires additional dependencies which can be installed into a virtualenv. A ``requirements.txt``
+file is provided with the necessary dependencies.
+
+Here is an example for building the HTML version of the devguide:
+
+.. code:: shell
+
+   $ python3 -m venv env
+   $ source env/bin/activate
+   (env) $ pip install -r requirements.txt
+   (env) $ make html
+
+After the processing completes, the new HTML documents will be available in the ``_build/html``
```

Comment:
Thanks @cjrh. I like the more explicit directions that you have for working with the devguide code. It will be a nice addition to the page Mariatta references. 👍